.github/workflows/test-cice.yml

```diff
@@ -74,6 +74,20 @@ jobs:
            ln -s ${GITHUB_WORKSPACE}/../CICE ${HOME}/cice
 #           ls -al ${HOME}/
 #           ls -al ${GITHUB_WORKSPACE}/
+      - name: check for tabs
+        run: |
+          cd $HOME/cice/cicecore
+          set cnt = 0
+          set ffiles = `find -P . -iname "*.f*"`
+          set cfiles = `find -P . -iname "*.c*"`
+          foreach file ($ffiles $cfiles)
+            set fcnt = `sed -n '/\t/p' $file | wc -l`
+            @ cnt = $cnt + $fcnt
+            if ($fcnt > 0) then
+               echo "TAB found: $fcnt $file"
```

Comment:
I'm a little confused about what this is doing.  It only affects testing with github actions?  If there are tabs in the source code, then github actions fails?  And this just flags where they are so they can be fixed?  (i.e. it doesn't fix them)